### ISSUE TITLE
在对OneKE进行lora微调时，input_ids、inputs与label_ids、labels一样。

### ISSUE THREAD

**Issue by syt1998** [zjunlp/IEPile#29]
请问，在对OneKE进行lora微调时，log显示inputs与labels都是train_file中的"output"字段：
input_ids:
[8853, 30988, 33711, 1115, 19997, 376, 30988, 33711, 33684, 30959, 1115, 19997, 376, 32129, 32520, 1115, 19997, 376, 32129, 32320, 1115, 19997, 376, 46856, 31391, 37159, 1115, 19997, 376, 33902, 1115, 5159, 29913, 2]
inputs:
 {"体征": [], "体征指标值": [], "其他检查": [], "其他治疗": [], "剂量或次数": [], "大小": []}</s>
label_ids:
[8853, 30988, 33711, 1115, 19997, 376, 30988, 33711, 33684, 30959, 1115, 19997, 376, 32129, 32520, 1115, 19997, 376, 32129, 32320, 1115, 19997, 376, 46856, 31391, 37159, 1115, 19997, 376, 33902, 1115, 5159, 29913, 2]
labels:
 {"体征": [], "体征指标值": [], "其他检查": [], "其他治疗": [], "剂量或次数": [], "大小": []}</s>
 请问这是什么原因以及如何解决呢？

lora微调脚本如下：
output_dir='/mnt/SYT/IE/Checkpoints/OneKE/Origin+Medical_Records_V'
mkdir -p ${output_dir}
CUDA_VISIBLE_DEVICES="1" nohup torchrun --nproc_per_node=1 --master_port=29500 /home/syt/IEPile-main/src/finetune.py \
    --do_train --do_eval \
    --overwrite_output_dir \
    --model_name_or_path '/mnt/SYT/Base_Models/OneKE' \
    --stage 'sft' \
    --model_name 'llama' \
    --template 'llama2_zh' \
    --train_file '/mnt/SYT/IE/Datasets/Medical_Records_V/Converted_Data/NER_Train_Data.json' \
    --valid_file '/mnt/SYT/IE/Datasets/Medical_Records_V/Converted_Data/NER_Valid_Data.json' \
    --output_dir=${output_dir} \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --gradient_accumulation_steps 256 \
    --preprocessing_num_workers 16 \
    --num_train_epochs 3 \
    --learning_rate 5e-5 \
    --max_grad_norm 0.5 \
    --optim "adamw_torch" \
    --max_source_length 2048 \
    --cutoff_len 2048 \
    --max_target_length 2048 \
    --save_strategy "epoch" \
    --save_total_limit 10 \
    --lora_r 64 \
    --lora_alpha 64 \
    --lora_dropout 0.05 \
    --bf16 \
    2>&1 | tee /home/syt/IEPile-main/Scripts/SFT/NER.out &

**Comment by guihonghao** [comment]
检查训练数据NER_Train_Data.json是否有问题。

**Comment by zxlzr** [comment]
请问您的问题解决了吗？

**Comment by syt1998** [comment]
已解决，谢谢。问题出在以下三参数的设置：
--max_source_length 2048
--cutoff_len 2048
--max_target_length 2048
